src/app-layer-dnp3.c

```diff
@@ -299,9 +299,11 @@ static uint16_t DNP3ProbingParser(Flow *f, uint8_t direction,
 
 end:
```

Comment:
What do you think of the code a few lines above coming to this label ?
```
if (DNP3ContainsBanner(input, len)) {
        SCLogDebug("Packet contains a DNP3 banner.");
        goto end;
    }
```

In this case, we certainly cannot use `hdr->control`